### PR TITLE
chore(deps): cleaned up irrelevant dependencies in ui-markdown-editor

### LINKED ISSUE #52: Dependencies Cleanup
Need to take a hard look at all our dependencies. 

This impacts security, development workflow (slow builds), and page load times.

### PR DESCRIPTION
**Pull request by Cronus1007**
# Closes #52 

### Changes
- Removed rimraf package
- Removed npm-run-all package

### Flags
- NPM documentation for rimraf https://www.npmjs.com/package/rimraf
- NPM documentation for npm-run-all https://www.npmjs.com/package/npm-run-all

### Related Issues
- Issue #52 

### Author Checklist
- [x] Ensure you provide a [DCO sign-off](https://github.com/probot/dco#how-it-works) for your commits using the `--signoff` option of git commit.
- [x] Vital features and changes captured in unit and/or integration tests
- [x] Commits messages follow [AP format](https://github.com/accordproject/techdocs/blob/master/DEVELOPERS.md#commit-message-format)
- [x] Extend the documentation, if necessary
- [x] Merging to `master` from `fork:branchname`
- [x] Manual accessibility test performed
    - [x] Keyboard-only access, including forms
    - [x] Contrast at least WCAG Level A
    - [x] Appropriate labels, alt text, and instructions

### CODE REVIEW

**Comment by Cronus1007** [comment]
@irmerk @mttrbrts Any updates?

File: packages/ui-markdown-editor/package.json [hunk 1]
```diff
@@ -72,15 +72,13 @@
...
     "lint-staged": "10.1.7",
-    "npm-run-all": "4.1.5",
     "postcss-preset-env": "6.7.0",
     "prop-types": "^15.7.2",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
     "react-scripts": "3.4.1",
     "react-test-renderer": "16.13.1",
     "recursive-copy": "2.0.10",
-    "rimraf": "3.0.2",
```
Comment (raghavdhingra) [inline packages/ui-markdown-editor/package.json:83]: @Cronus1007 Can you please specify the types of check that you've performed

**Review by shivaylamba — APPROVED** [review]

**Review by irmerk — CHANGES_REQUESTED** [review]
Needs investigation by @accordproject/maintainers-ui

**Comment by Cronus1007** [comment]
@irmerk I have rebase the commits. Any updates regarding this PR?

**Comment by irmerk** [comment]
Looking at this now @Cronus1007 , apologies for the delay. Can you rebase to resolve conflicts in `packages/ui-markdown-editor/package.json`?

**Comment by Cronus1007** [comment]
@irmerk I have resolved merge conflicts.

File: packages/ui-contract-editor/README.md [hunk 2]
```diff
@@ -116,7 +116,12 @@ clauseProps = {
 }
 ```
 
----
+The functions in `clauseProps` like `CLAUSE_DELETE_FUNCTION`,`CLAUSE_EDIT_FUNCTION` and `CLAUSE_TEST_FUNCTION`  are not implemented by default because they typically require interaction with other components or the persistence layer of the embedding application.
+These functions will be supplied by the user of the component.
+For more information in realtion to the `props` , `values` and `functionality` of Contract Editor visit [README.md](./src/ContractEditor/README.md)
```
Comment (irmerk) [inline packages/ui-contract-editor/README.md:121]: One last change, I think we should remove this last line since that `README` file no longer exists.

Comment (irmerk) [reply packages/ui-contract-editor/README.md:121]: Also keep the `---` horizontal rule just after this section you added.

**Comment by irmerk** [comment]
Still unsure about the huge addition to `package-lock` when dependencies are being removed. Please rebase to one commit.

**Comment by Cronus1007** [comment]
@irmerk I have updated a little bit of `ui-contract-editor/Readme.md` to resolve the newly linked issue.

**Pull request closed without merging**